packages/nimble-components/src/table/index.ts

```diff
@@ -114,11 +113,19 @@ export class Table<
     }
 
     protected childItemsChanged(): void {
-        if (!this.$fastController.isConnected) {
-            return;
-        }
+        void (async () => this.updateColumnsFromChildItems())().catch(ex => {
+            throw ex;
+        });
```

Comment:
Sorry, I wasn't thinking clearly in chat what the result would look like. This still results in an unhandled promise rejection (the throw inside the `catch` clause is converted to a promise) and ends up mixing Promise and async / await syntax unnecessarily. Lets just simplify to:

```suggestion
        void this.updateColumnsFromChildItems();
```

The `void` is enough of a hint that we explicitly don't care about responding to / recovering from the resulting Promise.